src/main/components/MockupName.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useContext } from 'react';
+import styled from 'styled-components';
+
+// Styles
+const Header = styled.header`
+  display: flex;
+  align-items: flex-end;
+  h3 {
+    margin-bottom: 0;
+  }
+`;
+
+const Name = styled.h3`
+  font-family: 'Halyard Text', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
Mentioned in person, but this displays as Times New Roman over here.  I honestly really liked the H3 font before but I'm not picky, but we do gotta make sure whatever fonts we use are included in the app.

Reply:
Yeah that’s really strange—not sure why it would do that here, since the rest of the stack should be okay on your machine. But agreed, either way we’ll bundled up the font when we launch.